### PR TITLE
Use futurize to fix has_key

### PR DESCRIPTION
**Pull request by persianpros**
Safe for py2 and py3, similar to https://github.com/OpenViX/enigma2/pull/673

It only changes has_key to "in" no other changes.

After this PR all PLi, ViX, Vision and ATV would be similar and without has_key.

### CODE REVIEW

File: lib/python/Components/Network.py [hunk 1]
```diff
@@ -557,7 +557,7 @@ def deactivateInterfaceFinished(self, extra_args):
 		(ifaces, callback) = extra_args
 
 		def checkCommandResult(iface):
-			if self.deactivateInterfaceConsole and self.deactivateInterfaceConsole.appResults.has_key("ifdown " + iface):
+			if self.deactivateInterfaceConsole and "ifdown " + iface in self.deactivateInterfaceConsole.appResults:
```
Comment (jbleyel) [inline lib/python/Components/Network.py:560]: this should be `
if self.deactivateInterfaceConsole and ("ifdown " + iface) in self.deactivateInterfaceConsole.appResults:`

or what you think?

Comment (persianpros) [reply lib/python/Components/Network.py:560]: You mean having parentheses around the check to make sure it checks both?

I agree with you but is the result any different

Comment (jbleyel) [reply lib/python/Components/Network.py:560]: yes .. i'ts the same result. I have test this.

Comment (persianpros) [reply lib/python/Components/Network.py:560]: So we're good :+1: 

I need to send this for other distros, this is the first time e2 scene is getting same style.

**Pull request merged**